.github/workflows/pull.yml

```diff
@@ -462,7 +462,7 @@ jobs:
     name: linux-jammy-py3-clang12-executorch
     uses: ./.github/workflows/_linux-test.yml
     needs: linux-jammy-py3-clang12-executorch-build
-    if: false # Has been broken for a while
+    if: true # Has been broken for a while
```

Comment:
Please just remove this line (i.e. `if: true` with the comment that something has been broken) sounds a bit weird, don't you think?
```suggestion
```